src/components/inline/Button/Button.module.css

```diff
@@ -1,9 +1,9 @@
 .button {
-  background-color: var(--color-brand);
-  border: 1px solid var(--color-brand);
+  background-color: var(--color-sheltertech-blue);
+  border: 1px solid var(--color-sheltertech-blue);
   color: var(--color-white);
   cursor: pointer;
-  font-family: var(--font-family-base);
+  font-family: var(--font-action);
   font-size: 14px;
   font-weight: 800;
   line-height: 1.5;
```

Comment:
You can't just use `var(--font-action)` on `font-family`, since `--font-action` contains the whole set of values that can be used for the [`font`](https://developer.mozilla.org/en-US/docs/Web/CSS/font) shorthand property. Instead, you can just replace all of these `font-*` properties with the single `font` shorthand property.